packages/Search/src/details/DTK_DetailsKokkosExt.hpp

```diff
@@ -80,14 +80,14 @@ int clz( uint32_t x )
 
 //! Compute the maximum of two values.
 template <typename T>
-KOKKOS_INLINE_FUNCTION T max( T a, T b )
+KOKKOS_INLINE_FUNCTION T const &max( T const &a, T const &b )
```

Comment:
@sslattery @aprokop I pushed these changes.  It was an oversight on my part.

Reply:
Good catch.